docs/superpowers/specs/2026-05-12-user-device-fcm-token-design.md

```
@@ -0,0 +1,400 @@
+# 디바이스 정보 및 FCM 토큰 저장 기능 설계
+
+- **작성일**: 2026-05-12
+- **관련 이슈**: [#23 기능추가][인증] 디바이스 정보 및 FCM 토큰 저장 기능 추가
```

markdownlint-cli2 (0.22.1)

[warning] Reference links and images should use a label that is defined
Missing link or image reference definition: "인증"

(MD052, reference-links-images)